style.css

```diff
@@ -195,6 +195,7 @@ body {
   background-color: white;
   justify-content: center;
   width: 70vw;
+  height: 100vh;
```

Comment:
- [ ] you can adjust the margin of the card so it can be displayed correctly, right now it hiding the nav-bar.

![1](https://user-images.githubusercontent.com/61562626/136550709-94ae1612-1fc6-4901-8dc1-294d983b5291.jpeg)